### ISSUE TITLE
install2.r doesn't permit multiple arguments to repo

### ISSUE THREAD

**Issue by cboettig** [eddelbuettel/littler#8]
Can't figure out how to get docopt to allow me to specify multiple repos in the argument to repos.   

Mostly this is really annoying when using some custom repo that also depends on some CRAN packages -- since the custom argument overrides the default repo these dependencies can't install automatically.  

Seems like this should be a simple thing to fix but for some reason I'm a bit stumped on how to go about it.

**Comment by eddelbuettel** [comment]
I think I finally have something better, thanks to [drat](http://dirk.eddelbuettel.com/drat.html):

```{.bash}
edd@max:~/git/littler(master)$ cat ~/.littler.r

## add RStudio CRAN mirror
drat:::add("CRAN", "http://cran.rstudio.com")

## add Dirk's drat
drat:::add("eddelbuettel")

edd@max:~/git/littler(master)$ r -pe'getOption("repos")'
                                 CRAN                          eddelbuettel 
            "http://cran.rstudio.com" "http://eddelbuettel.github.io/drat/" 
edd@max:~/git/littler(master)$ 
```

I made the corresponding changes in the the `examples/` directory of littler.  Now rocker would just have to `echo` into, say, `/etc/littler.r` and we'd be set.  

What do you think?

**Comment by cboettig** [comment]
Nice!

**Comment by eddelbuettel** [comment]
Making my day, five characters at a time ;-)

I tested that some more here, and I do think it is the way to go.  If you don't object that is.

**Comment by eddelbuettel** [comment]
This can be closed as we have the drat solution and #12 to support multiple -r uses.
